persistence-jpa/javadsl/src/main/java/com/lightbend/lagom/internal/javadsl/persistence/jpa/JpaReadSideImpl.java

```diff
@@ -218,14 +218,14 @@ private Query postgresqlBindUpdateOffsetQuery(Query query, Long sequenceOffset,
                     .setParameter(10, tag.tag());
         }
 
-        private Query defaultBindUpdateOffsetQuery(Query query, Long sequenceOffset, String timeUuidOffset) {
+        private Query defaultBindUpdateOffsetQuery(JdbcProfile profile, Query query, Long sequenceOffset, String timeUuidOffset) {
             // H2:
             // merge into "read_side_offsets" ("read_side_id","tag","sequence_offset","time_uuid_offset")
             //   values (?,?,?,?)
             return query
                     .setParameter(1, readSideId)
                     .setParameter(2, tag.tag())
-                    .setParameter(3, sequenceOffset)
+                    .setParameter(3, (profile instanceof OracleProfile && sequenceOffset == null) ? "" : sequenceOffset)
```

Comment:
It's a single way, what I found without using internal Hibernate API.

Reply:
Tried to use next implementation:
```java
Parameter<Long> seqOffsetParam = new Parameter<Long>() {
    @Override
    public String getName() {
        return null;
    }

    @Override
    public Integer getPosition() {
        return 3;
    }

    @Override
    public Class<Long> getParameterType() {
        return Long.class;
    }
};
return query
        .setParameter(1, readSideId)
        .setParameter(2, tag.tag())
        .setParameter(seqOffsetParam, sequenceOffset)
        .setParameter(4, timeUuidOffset);
```
But it not help. All other approaches using internal Hibernate API.